Mathlib/Topology/Baire/BaireMeasurable.lean

```diff
@@ -43,6 +43,15 @@ scoped[Topology] notation3 "∃ᵇ "(...)", "r:(scoped p => Filter.Frequently p
 
 variable {α}
 
+theorem coborder_mem_residual {s : Set α} (hs : IsLocallyClosed s) : coborder s ∈ residual α :=
+  residual_of_dense_open hs.isOpen_coborder dense_coborder
+
+theorem closure_residualEq {s : Set α} (hs : IsLocallyClosed s) : closure s =ᵇ s := by
+  rw [Filter.eventuallyEq_set]
+  filter_upwards [coborder_mem_residual hs] with x hx
+  nth_rewrite 2 [← closure_inter_coborder (s := s)]
+  simp [hx]
```

Comment:
I failed to find a better home for these 2 lemmas. I assume that having them here is better than having them as `have`s in a lemma.

Reply:
I think this is an okay place, even if not absolutely ideal.